src/sot_gepetto_viewer/graph.py

```diff
@@ -125,11 +199,13 @@ def _edgeEntitySOT(self, s):
         for t,n in zip(tasks, nodes):
             if not self.nodes.has_key(t):
                 self._createGraphBackwardFromEntity(t)
-            e = self.graph.addEdge (self.nodes[t], n, "error")
-            # TODO errorTimeDerivative
-            n = "sot_" + s + "/task_" + t + "/error"
-            self.edges[n] = (t, e)
-            self.edgesBack[e] = n
+            for i in self.filter:
+                if i == "0":
+            	    e = self.graph.addEdge (self.nodes[t], n, "error")
+                    # TODO errorTimeDerivative
```

Comment:
You have a weird indentation here.

Reply:
One tabulation at the start of the line caused the indentation problem